sdk/cosmos/azure-spring-data-cosmos/README.md

```diff
@@ -296,11 +297,14 @@ Azure spring data cosmos supports specifying annotated queries in the repositori
 
 ```java
 public interface AnnotatedQueriesUserRepositoryCodeSnippet extends CosmosRepository<User, String> {
-    @Query(value = "select * from c where c.firstName = @firstName and c.lastName = @lastName")
+    @Query("select * from c where c.firstName = @firstName and c.lastName = @lastName")
```

Comment:
Since you have updated code snippets, you also need to update the readme with correct line numbers for these code snippets. That is the reason, Build Analyze CI is failing.

Reply:
ah, didn't realize how this was wired up.  i updated the line numbers in the readme as well as the snippets, will keep an eye on the build

Reply:
@kushagraThapar i'm not sure what's wrong - the code snippets are exactly the same as the readme and the line numbers are correct, as far as i can tell, but it's still failing on the "verify-readme-codesnippet" validation.  any ideas?

Reply:
`UserRepository` is modified in Readme, but not in the java file. So we need to update the `UserRepository` java file, and then update the readme with correct line numbers. 
I know this is annoying to update the same thing at two places, I will see how we can resolve this going forward.

Reply:
thank you!  i was looking at AnnotatedQueriesUserRepositoryCodeSnippet and AnnotatedQueriesUserReactiveRepositoryCodeSnippet and somehow completely missed UserRepository.